src/main/pages/che-7/overview/proc_finding-che-cluster-url-using-openshift-4-cli-tools.adoc

```diff
@@ -20,3 +20,15 @@ time="2019-07-12T09:50:13Z" level=info msg="Custom resource eclipse-che updated"
 time="2019-07-12T09:50:13Z" level=info msg="{prod} is now available at: http://che-che.apps.cluster-fre-f0a2.fre-f0a2.openshiftworkshop.com"
 time="2019-07-12T09:50:13Z" level=info msg="Updating eclipse-che CR with version: 7.0.0-RC-2.0"
 ----
+endif::[]
+ifeval::["{project-context}" == "crw"]
+* Wait for the logs to show the message `{prod} is now available at:` followed by {prod-short} URL:
++
+[subs="+quotes,+attributes",options="nowrap"]
+----
+time="2019-07-12T09:50:13Z" level=info msg="Updating eclipse-che CR with {prod} server URL: http://codeready-workspaces.apps.cluster-fre-f0a2.fre-f0a2.openshiftworkshop.com"
```

Comment:
is this block really needed? If it's the same in upstream too, then you could probably just define a new variable called `{deployment-route}` and set it to `che-che` or `codeready-workspaces` based on value of `{project-context}` variable.